docs/pages.yml

```diff
@@ -10,15 +10,19 @@
     - HPC Cluster: 'content/getting_started/cluster.md'
     - Virtual Machine: 'content/getting_started/virtual_machine.md'
     - Manual Installation: 'content/getting_started/manual_installation.md'
-- Documentation:
-    - Create an App: 'content/getting_started/create_an_app.md'
-    - Framework:
-        - Systems: !include content/framework/systems/pages.moose.yml
-    - Modules:
-        - Phase Field:
-            - Systems: !include content/modules/phase_field/systems/pages.moose.yml
-        - Contact:
-            - Systems: !include content/modules/contact/systems/pages.moose.yml
-    - Documenting Applications:
-        - Creating Documentation: 'content/doc_generation/Overview.md'
-        - Moose Flavored Markdown: 'content/doc_generation/MooseFlavoredMarkdown.md'
+- Application Development:
+    - Create an Application: 'content/application_development/create_an_app.md'
+- Framework:
+    - Systems: !include content/framework/systems/pages.moose.yml
+- Modules:
+    - Phase Field:
+        - Systems: !include content/modules/phase_field/systems/pages.moose.yml
+    - Contact:
+        - Systems: !include content/modules/contact/systems/pages.moose.yml
+- Utilites:
+    - Documentation:
+        - Creation: 'content/utilities/documentation/create.md'
+        - Moose Flavored Markdown: 'content/utilities/documentation/moose_flavored_markdown.md'
+- Help:
+    - Frequently Asked Questions: 'content/help/faq.md'
+    - Contact Us: 'content/help/contact_us.md'
```

Comment:
What if rename "Application Development" to something shorter: "App Development"